docs/release.md

```diff
@@ -73,19 +71,20 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 5. A new `v0.85.0` release should be automatically created on Github by now. Edit it and use the contents from the CHANGELOG.md as the release's description.
 
 ## Producing the artifacts
-
 The last step of the release process creates artifacts for the new version of the collector and publishes images to Dockerhub. The steps in this portion of the release are done in the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repo.
 
-1. Update the `./distribution/**/manifest.yaml` files to include the new release version.
+1. Update the `./distributions/**/manifest.yaml` files to include the new release version.
 
 2. Update the builder version in `OTELCOL_BUILDER_VERSION` to the new release in the `Makefile`. While this might not be strictly necessary for every release, this is a good practice.
 
-3. Create a pull request with the change and ensure the build completes successfully. See [example](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/71).
+3. Create and push a new branch with this change, and apply the tag `cmd/builder/v0.xxx.0` to match the version number used above by running the command `make push-tags TAG=cmd/builder/v0.xxx.0`. This will build and release the ocb binaries under the opentelemetry-collector-releases distribution and is required to be completed **before** creating the pull request to merge the branch in the next step below. Building the distributions relies on using the latest ocb binary and if this workflow does not complete successfully, the rest of the build/release actions may not complete successfully.
```

Comment:
Is there any way we could specify the dependency in the workflows instead of requiring an additional tag? The release process is already fairly complex, it would be nice not to add any new steps.

Reply:
I am open to suggestions and wanted to make this happen, but after looking at the release workflow (build on PR, build again on merge, then release on tag), I couldn't figure out a good way to separate the dependencies. Especially since the makefile simply downloads the prebuilt ocb release to build the different distributions. 
The other way to approach this might be using a token with cross-repo access to build the ocb release in the opentelemetry-collector repo but push it to the releases repo, but I wasn't certain if that was best practice security-wise and might make it more opaque which release is happening where.

Reply:
>Especially since the makefile simply downloads the prebuilt ocb release to build the different distributions.

This is a "dumb" approach, but here's how I would do it:
* Run something like `go build  go.opentelemetry.io/collector/cmd/builder@v0.106.0 -o ./bin/ocb` to build the binary.
* Use the `cache` action to cache the binary using the tag version (and maybe OS+arch) as the cache key.
* Release OCB using the binaries from the cache (or just pull them from the build results).
* Use the cached binary to build the Collector.

We would just need to link the workflows to ensure that OCB is built before the Collector. If necessary, I'd be okay migrating our separate workflows to jobs in a "Release" workflow to allow us to just link the jobs.

>Another option (not without its downsides) is to decouple the OCB release from the Collector distros release, and use OCB v.(N-1) to build Collector v.N.

I'd be okay with this as a backup if we can't do something like the above. It would be nice to continue to build with the latest OCB to let us take advantage of Collector API updates without waiting for an additional release if we can make it work.

Reply:
And running `go build` would only rely on the source code release that was generated in the `opentelemetry-collector` repository?

Reply:
>And running `go build` would only rely on the source code release that was generated in the `opentelemetry-collector` repository

It should, yeah. So long as you point it at the module, it will only use whatever we publish for that module.

>I wasn't certain if this was something that was handled by goreleaser, which in that case, would need to be run prior to building the collector distributions.

I'm still not sure how to point Goreleaser at a remote module, but at a minimum we should be able to do something similar to the approach you took in the -releases PR to just build it twice.

Reply:
we publish the source code and corresponding go module here, and that gets pulled by the build script in releases. was able to publish a release in both forked repositories under jackgopack4/opentelemetry-collector and jackgopack4/opentelemetry-collector-releases

Reply:
@evan-bradley any further thoughts/concerns on this PR or the releases PR? Thanks.

Reply:
Sorry for the delay, I was digging into it to see if there's a better way to approach building the source, since I'm also looking to add a binary to be released from -releases. I think that part is good to go, but I left a comment on the tagging I just want to make sure we're being deliberate about. I'll resolve this thread since I think the approach to building the binary is as good as we can do.